nikola/conf.py.in

```diff
@@ -140,7 +140,7 @@ TIMEZONE = ${TIMEZONE}
 # Note that this does not affect DATE_FORMAT.
 # FORCE_ISO8601 = False
 
-# Date format used to display post dates.
+# (Translatable) Date format used to display post dates.
```

Comment:
Changed that. There are some other places (unrelated to my edits) where it still is at the beginning of the line.